plugins/gradle/gradle.plugin.zsh

```diff
@@ -60,6 +60,35 @@ _gradle_does_task_list_need_generating () {
   [[ ! -f .gradletasknamecache ]] || [[ build.gradle -nt .gradletasknamecache ]]
 }
 
+##############
+# Parse the tasks from `gradle(w) tasks --all` into .gradletasknamecache
+# All lines in the output from gradle(w) that are between /^-+$/ and /^\s*$/
+# are considered to be tasks. If and when gradle adds support for listing tasks
+# for programmatic parsing, this method can be deprecated.
+##############
+_gradle_parse_tasks_to_gradletasknamecache () {
```

Comment:
I would also change this function name to `_gradle_parse_tasks`. See [my comment below](https://github.com/robbyrussell/oh-my-zsh/pull/5230/files#r78707652).